src/containerapp/azext_containerapp/_params.py

```diff
@@ -424,3 +424,19 @@ def load_arguments(self, _):
         c.argument('workload_profile_type', help="The type of workload profile to update. Run 'az containerapp env workload-profile list-supported -l <region>' to check the options for your region.")
         c.argument('min_nodes', help="The minimum node count for the workload profile")
         c.argument('max_nodes', help="The maximum node count for the workload profile")
+
+    # Patch
+    with self.argument_context('containerapp patch list') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
+        c.argument('managed_env', options_list=['--environment', '-e'], help='Name or resource id of the Container App environment.')
+        c.argument('show_all', help='Show all patchable and unpatchable container apps')
+
+    with self.argument_context('containerapp patch apply') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
```

Comment:
If these three arguments are applied to all commands in `az containerapp patch`, you could define argument context for `containerapp patch` once, and it will applied to all commands in `containerapp patch`

Reply:
@Juliehzl, could you please clarify if the following is the suggested change and we can safely remove registering arguments separately for `az containerapp patch list`, `az containerapp patch apply` and `az containerapp patch interactive`?

```
 with self.argument_context('containerapp patch') as c:
        c.argument('resource_group_name', arg_type=resource_group_name_type)
        c.argument('managed_env', options_list=['--environment', '-e'], help='Name or resource id of the Container App environment.')
        c.argument('show_all', help='Show all patchable and unpatchable container apps')
```

Reply:
Updated. Made the change from my previous comment.